### ISSUE TITLE
#import <React/RCTEventEmitter.h> not found

### ISSUE THREAD

**Issue by maystrenkoYurii** [react-native-community/react-native-push-notification-ios#32]
![Снимок экрана 2019-10-08 в 14 55 28](https://user-images.githubusercontent.com/23016311/66393622-b547c500-e9db-11e9-9993-25e1a3312380.png)

**Comment by say95** [comment]
Hey @maystrenkoYurii , I am facing the same issue. Are you able to resolve this issue?

**Comment by opiruyan** [comment]
What is your RN version? 61? Does your Podfile conform to required?

**Comment by guopeng1994** [comment]
same issues in RN 0.59.9

**Comment by BrandonMHogan** [comment]
same issue using expo 35.0.0

**Comment by muriloribeiro34** [comment]
use --> cd ios --> pod install

**Comment by GregoryBabonaux** [comment]
Same issue with RN 0.59.10

i tried pod install with no success

**Comment by muriloribeiro34** [comment]
pod install so it works in react-native 0.60+, sorry

**Comment by GregoryBabonaux** [comment]
> pod install so it works in react-native 0.60+, sorry

Yep, it probably work for RN 0.60+, but the major issues related here are for RN 0.59.x

**Comment by caner-karaman** [comment]
Same issue after upgrading RN 0.61.3

**Comment by cx5168** [comment]
RN0.61.4 same issue!!!

![スクリーンショット 2019-11-22 15 51 10](https://user-images.githubusercontent.com/29448456/69410805-21de0f80-0d4f-11ea-9131-f1c8c881e4a9.png)

**Comment by cx5168** [comment]
I m solved it!

1. Libraries→PushNotificationIOS.xcodeproj→build setting→Search Paths→Header Search Paths:
$(SRCROOT)/../../../../ios/Pods/Hearders/Public

2. menu(Product)→Scheme→Edit Scheme:
 [+]→Pods→React
![123](https://user-images.githubusercontent.com/29448456/69602688-c4520780-105b-11ea-882d-76013d99edc0.png)

3. clean→build

**Comment by Naturalclar** [comment]
@maystrenkoYurii has this issue been solved for you?
If so, I'll go ahead and close this issue :)

**Comment by ShawnKuenzlerZEN** [comment]
Typo in your path for "Headers" but it worked for me. Thanks.
